packages/react/src/components/List/List.mdx

```diff
@@ -99,6 +91,22 @@ const yankees = {
 | emptyState                 | node, string                                                                                                                                         | 'No list items to show'                         | content shown if list is empty                                                                                                                                                                                                                                                                                                |
 | handleLoadMore             | function                                                                                                                                             |                                                 | callback if a load more row is clicked called with the id of the row id.                                                                                                                                                                                                                                                      |
 
+### ListItemPropTypes
+
+| Name                   | Type                             | Default | Description                                                     |
+| :--------------------- | :------------------------------- | :------ | :-------------------------------------------------------------- |
+| children               | arrayOf(object)                  |         | Array of child items if the item is a parent                    |
+| content                | shape                            |         |                                                                 |
+| content.icon           | node                             |         | The icon to show for each item                                  |
+| content.rowActions[]   | oneOfType([arrayOf(node), func]) |         | The row actions for each item                                   |
```

Comment:
We should define a deprecated designation for our docs, so it's clear when something is deprecated like the arrayOf(nodes) here. Maybe the † (<kbd>⌥</kbd>+<kbd>t</kbd>) since it won't effect the layout of the markdown in anyway?

Reply:
Yea, up until now we have just added "Deprecated" in the description or in the type column (which seems a bit weird). I'll add it to description here.